server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/ConsistencyTasksIntegrationTest.java

```diff
@@ -281,9 +308,11 @@ void shouldRecomputeQuotas(GuiceJamesServer server) throws Exception {
                     .times(1)
                     .whenQueryStartsWith(updatedQuotaQueryString));
 
-        probe.appendMessage(BOB.asString(), inbox,
-            new ByteArrayInputStream("Subject: test\r\n\r\ntestmail".getBytes(StandardCharsets.UTF_8)), new Date(),
-            !IS_RECENT, new Flags(Flags.Flag.SEEN));
+        probe.appendMessage(BOB.asString(), inbox, AppendCommand.builder()
+            .withFlags(FLAGS)
+            .withInternalDate(DATE)
+            .notRecent()
```

Comment:
Here we don't care about the DATE, & recent, only about the flags, right?

Reply:
BTW we can use the APPEND_COMMAND constant

Reply:
Funny thing is if i use the constant here and try to test all cases, this test will keep running forever. Runs fine on its own though. Not sure why so i left it out on purpose.

Reply:
Clenup issue between tests perhaps?

Reply:
AppendCommand is stateful as it depends on an InputSTream.

The APPEND_COMMAND should maybe be a `Supplier<AppendCommand>` to avoid this trap.